ruby/constants/menuOptions.rb

```diff
@@ -0,0 +1,14 @@
+@menuOptions = {
```

Comment:
The `@` symbol is used for instance variables, so is unnecessary here (and would actually have some interesting side effects - I think this is declaring an instance variable on the scope in which this is executing, which is itself an object).